src/cases/containers/restricted-case-access-container/restricted-case-access-container.component.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+import { of } from 'rxjs';
+import { CASEWORKERS } from '../../../../api/test/pact/constants/work-allocation/caseworkers.spec';
+import { CASEROLES } from '../../../../api/workAllocation/constants/roles.mock.data';
+import { CaseReferencePipe } from '../../../hearings/pipes/case-reference.pipe';
+import { AllocateRoleService } from '../../../role-access/services';
+import { CaseworkerDataService, WASupportedJurisdictionsService } from '../../../work-allocation/services';
+import { RestrictedCaseAccessContainerComponent } from './restricted-case-access-container.component';
+
+describe('RestrictedCaseAccessContainerComponent', () => {
+  let component: RestrictedCaseAccessContainerComponent;
+  let fixture: ComponentFixture<RestrictedCaseAccessContainerComponent>;
+  const mockAllocateService = jasmine.createSpyObj('AllocateRoleService', ['getCaseAccessRolesByCaseId']);
+  const mockWASupportedJurisdictionsService = jasmine.createSpyObj('WASupportedJurisdictionsService', ['getWASupportedJurisdictions']);
+  const mockCaseworkerDataService = jasmine.createSpyObj('CaseworkerDataService', ['getCaseworkersForServices']);
+  const mockActivatedRoute = {
+    snapshot: {
+      params: {
+        cid: '1234123412341234'
+      }
+    }
+  };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [],
+      declarations: [
+        RestrictedCaseAccessContainerComponent,
+        CaseReferencePipe
+      ],
+      providers: [
+        { provide: AllocateRoleService, useValue: mockAllocateService },
+        { provide: WASupportedJurisdictionsService, useValue: mockWASupportedJurisdictionsService },
+        { provide: CaseworkerDataService, useValue: mockCaseworkerDataService },
+        { provide: ActivatedRoute, useValue: mockActivatedRoute }
+      ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
```

Comment:
@johnbenjamin-hmcts - is there a reason why there are two `beforeEach()` blocks? Can they be combined?

Reply:
Thank you @Daniel-Lam, I have done the changes as you suggested.